src/plays/states/States.jsx

```diff
@@ -0,0 +1,101 @@
+import { useState } from "react";
+import { useLocation } from "react-router-dom";
+import { getPlayById } from "meta/play-meta-util";
+import PlayHeader from "common/playlists/PlayHeader";
+import "./states.css";
```

Comment:
Actual file name us still with capital case. It is failing the build @Abhishek-90